onboarding/scaffolding.py

```diff
@@ -232,12 +241,54 @@ def perform_mutate(cls, form, info: ResolveInfo):
         return cls.perform_mutate_for_anonymous_user(form, info)
 
 
+def _migrate_onboarding_info_to_scaffolding(request):
```

Comment:
If we anticipate deprecating this function after we fully get rid of OnboardingStep1V2Info, let's add a note to that effect.

Reply:
Good idea, just did that. Also renamed the function and contextualized the docs in light of #2156, which we can hopefully merge shortly after this PR!